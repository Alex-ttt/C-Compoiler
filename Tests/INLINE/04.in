inline void fo(int arg){
	arg = 5;
}

void main(){
	const int a;
 	fo(a);
}