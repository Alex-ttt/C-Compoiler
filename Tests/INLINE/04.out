 Symbols Table:
   fo  void function fo(
          arg int
  )
    {
      Statements:
        arg
      =
        5
--------------------------
      Symbols Table:
    }
   main  void function main(
  )
    {
      Statements:
      {
        Statements:
          param0
        =
          a
--------------------------
          param0
        =
          5
--------------------------
      BLOCK_LABEL2:
        Symbols Table:
      }
--------------------------
--------------------------
      Symbols Table:
              a const int
    }
